TREP-LABS/realdrip-fronend
# #168363064 User(hospital admin) signup

#### What does this PR do?
- Implements user(hospital admin) signup feature

#### What are the relevant pivotal tracker stories?
[#168363064 ](https://www.pivotaltracker.com/story/show/168363064 )

#### How should this be manually tested?
- Setup the app locally
- Check out to the branch containing the changes of this PR
- Start the application
- The backend app needs to be running to be able to test this feature
- When you visit `http://localhost:<port>/signup`, where `port` is the port number the app is running, you should see the signup page
- There you can test the signup feature which should work within the scope of its acceptance criteria

#### Screenshots 
The signup page:  
![download (1)](https://user-images.githubusercontent.com/26048536/65496977-59e0e780-deb1-11e9-9fe1-e069dc16e535.png)
